egs/wsj/s5/steps/nnet3/chain/e2e/prepare_e2e.sh

```diff
@@ -21,6 +21,9 @@ type=mono             # can be either mono or biphone -- either way
 ci_silence=false      # if true, silence phones will be treated as context independent
 
 scale_opts="--transition-scale=0.0 --self-loop-scale=0.0"
+tie=false
```

Comment:
more documentation please!  (Also make sure any important options are mentioned in the usage message).